.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -48,12 +49,24 @@ runs:
           run: |
               yum install -y gcc pkgconfig openssl openssl-devel which curl redis6 gettext --allowerasing
 
+        - name: Install software dependencies for Ubuntu MUSL
+          shell: sh
+          if: "${{ inputs.os == 'ubuntu-latest-musl' }}"
+          run: |
+              apk update
+              wget -O - https://sh.rustup.rs | sh -s -- -y
+              source "$HOME/.cargo/env"
+              apk add protobuf-dev musl-dev make gcc
```

Comment:
keep in mind that below we install specific version of protobuf

Reply:
You're right.
Tried to fix it,  but unfortunately the version `25.1` doesn't exists in the alpine repo,
latest version is `24.4-r1`.
The official protobuf repo doesn't seem to have a compiled `musl` package,  so I guess we need to rely on `24.4-r1`.   (or compile it ourself).  
Since the binary is being compiled successfully with `24.4-r1`,   do you still think it's crucial to use 25.1?